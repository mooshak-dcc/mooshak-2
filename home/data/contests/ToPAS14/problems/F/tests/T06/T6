515 5 9
AFFFAFFFF
AVVVVVXVA
AVVVIVXVA
AVVVVVTVF
AAAAAAAAA
5
E6
N2
W3
N6
N1
