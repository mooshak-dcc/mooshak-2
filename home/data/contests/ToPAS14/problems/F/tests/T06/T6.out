Dentro(4,2):511:ND
